db/migrate_cards/data/metric_stuff_5_from_dev.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
I really want to start looking for more opportunities to put pretty much anything that we can't sensibly read here into code.  (Even the readable stuff is hard to comment on here without diffs).

Nothing to change for this PR, but soon...